### ISSUE TITLE
This project could be made into a elm package.

### ISSUE THREAD

**Issue by nicu-chiciuc** [garrickb/elm-voronoi#1]
Great project. I was looking for something like this for the past few days. You should definitely think about extracting the main parts and publishing it as an elm package.

**Comment by shamansir** [comment]
Yes, please make it a separate package, it would be very useful.

**Comment by garrickb** [comment]
Thanks for the support, but I don't think my implementation is done efficiently enough for me to publish for others to use. I think it could serve as a resource for people who want to implement it better, though!

It was a just a fun project that I wanted to use in some game development, but never had time to actually use 😄
